### ISSUE TITLE
Accessibility Tree (TreeView Pane) always displays as empty? [aViewer version run from WAT 2015]

### ISSUE THREAD

**Issue by jgauvreau** [ThePacielloGroup/aviewer#23]
I have the 2015-05-20 version of WAT installed in IE 11 on my Windows 7 SP1 64-bit laptop. When I run aViewer from WAT no content displays in the accessibility tree (TreeView) pane of aViewer. The TreeList and CodeEdit panes work fine. My colleagues are having the same problem. 

I saw some other posts stating others experienced problems with Windows 7 64-bit and aViewer. Are there any possible solutions/workarounds yet?

![image](https://cloud.githubusercontent.com/assets/12006767/22315382/5775dbde-e334-11e6-87c2-1eea8b43eca3.png)

**Comment by patrickhlauke** [comment]
WAT is not in active development anymore, and the version of aViewer that comes with it is outdated. I'd suggest installing the standalone aViewer https://github.com/ThePacielloGroup/aviewer/releases and seeing if the problem persists there too.

**Comment by jgauvreau** [comment]
Thanks for letting me know that the version of aViewer in WAT is outdated. I tried downloading and installing the standalone aViewer 2.0 version but I seem to have the same issue where the TreeView pane is empty. The bigger problem is that the standalone version keeps crashing. I seem to be having the same experience documented by another user in unresolved, but closed issue #20. Any other tips to try? Is there a manual way I can check if the DLL is registered without using the UI?

**Comment by ghost** [comment]
Could you try this version? https://github.com/ThePacielloGroup/aviewer/tree/Develop/package

**Comment by jgauvreau** [comment]
Thanks! I downloaded the 64-bit .exe from the path above (I wasn't sure if I needed to download any of the other files in the package.) The good news is when I run the new .exe version I now see the accessibility tree pane populated. Thanks for finding a solution for my issue on Windows 7!

The only related bug that seems to still exist for the standalone version of aViewer is that whenever I try to show/hide the accessibility pane aViewer becomes non-responsive and I have to manually close the program out and restart it.

![image](https://cloud.githubusercontent.com/assets/12006767/22475693/c632517e-e7ad-11e6-986f-3212df5234cb.png)

**Comment by ghost** [comment]
I have updated aViewer.
new 64bit exe file https://github.com/ThePacielloGroup/aviewer/blob/Develop/package/aViewer64bit.exe

**Comment by jgauvreau** [comment]
Jun, thanks for the updated version of aViewer. I tried it out yesterday and the good news is it didn't crash for me when I tried to hide the accessibility tree pane using the mouse on the button or via the menu. 

Unfortunately it did still crash if I:
1. Tabbed to the << button and activated it with the spacebar. 
2. Tried to change any settings (see screenshot below)
![image](https://cloud.githubusercontent.com/assets/12006767/22839790/ea3b7f08-ef98-11e6-9a29-8904b1baa8c2.png)

Also, once I closed the left TreeView pane I couldn't get it to open again (unless I closed out of the application and reopened aViewer). The issue with the TreeView pane being "stuck" closed happens if I manually use the mouse to click on the "Collapse (or Expand) Button for TreeView" as well as if I use the View > Collapse > TreeView menu option. 
NOTE: I am able to repeatedly expand/collapse the two other panes (TreeList and CodeEdit) without a problem using either the mouse or the menu options.

Screenshot showing TreeView pane collapsed (menu option still available) but cursor to drag open pane not functioning:
![image](https://cloud.githubusercontent.com/assets/12006767/22748724/fc092434-edf8-11e6-8f8d-139140f3dc5f.png)

Scrrenshot showing TreeView pane being adjustable (using draggable cursor the right of the expand/collapse button) after restarting aViewer:
![image](https://cloud.githubusercontent.com/assets/12006767/22748901/9c40d834-edf9-11e6-9b8d-c7b6fe7fcd72.png)

**Comment by jgauvreau** [comment]
I wanted to follow-up on this issue. I wondered if you will be publishing the revised version of aViewer (new 64bit exe file https://github.com/ThePacielloGroup/aviewer/blob/Develop/package/aViewer64bit.exe) as an official new minor version?

**Comment by ghost** [comment]
I cannot reproduce it yet on a virtual machine.
However, I built a new window 7 PC. Please give me some time.

**Comment by patrickhlauke** [comment]
Latest version of aViewer (2.0.1.0) just released https://github.com/ThePacielloGroup/aviewer/releases/tag/2.0.1.0

@jgauvreau could you uninstall your current one, do a fresh install of this one, and let us know if the issue still persists?

**Comment by jgauvreau** [comment]
@patrickhlauke @jun325 
I just uninstalled and reinstalled the new version on my Windows 7 machine. The secondary issues that I reported above are still occurring. However, for ease of tracking I'm think that since the original issue I reported (about the Accessibility Tree not showing up at all) is fixed we could close this issue and I'll just log new separate issues for the problems that are still occurring.
